qiskit/algorithms/optimizers/__init__.py

```diff
@@ -122,6 +122,15 @@
    ESCH
    ISRES
 
+Other Optimizers
+==================
+
+.. autosummary::
+   :toctree: ../stubs/
+   :nosignatures:
+
+    SuffixAveragingOptimizer
+
```

Comment:
Could we just put that into the existing section? "Other optimizers" is not very descriptive, so I think we can just keep it with the others 🙂

Reply:
Which section is appropriate? This can become both local and global optimizer. `Optimizer Base class` might be good.